js/views/pageNavVw.js

```diff
@@ -501,11 +501,13 @@ module.exports = baseVw.extend({
   },
 
   navBackClick: function(){
-    window.history.back();
+    history.action = 'back';
```

Comment:
I see this is why you added parameters to the history object. That seems like it could cause surprises down the road. Is there a way to tell if the new route being executed is going back inside the execute hook in the router?